ui/app/components/app/permission-page-container/index.scss

```diff
@@ -288,3 +153,126 @@
     }
   }
 }
+
+.permission-result {
+  @extend %header--24;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+  color: $Black-100;
+  height: 114px;
+
+  &__content {
+    display: flex;
+    justify-content: center;
+    width: 228px;
+    padding: 10px;
+  }
+
+  &__icons {
+    display: flex;
+  }
+
+  &__center-icon {
+    display: flex;
+    position: relative;
+    justify-content: center;
+    align-items: center;
+    font-size: 12px;
+  }
+
+  h1 {
+    font-size: 14px;
+    line-height: 18px;
+    padding: 8px 0 0;
+  }
+
+  h2 {
+    font-size: 12px;
+    line-height: 17px;
+    color: #6A737D;
+    padding: 0;
+  }
+
+  &__check {
+    width: 40px;
+    height: 40px;
+    background: white url("/images/permissions-check.svg") no-repeat;
+    position: absolute;
+  }
+
+  &__reject {
+    position: absolute;
+    background: white;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    i {
+      color: #D73A49;
+      transform: scale(3);
+    }
+  }
+
+  &__identicon, .icon-with-fallback__identicon {
+    width: 32px;
+    height: 32px;
+
+    &--default {
+      background-color: #777A87;
+      color: white;
+      width: 64px;
+      height: 64px;
+      border-radius: 32px;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      font-weight: bold;
+    }
+  }
+
+  &__identicon-container, .icon-with-fallback__identicon-container {
+    height: auto;
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 64px;
+    width: 64px;
+  }
+
+  &__identicon-border, .icon-with-fallback__identicon-border {
+    height: 64px;
+    width: 64px;
+    border-radius: 50%;
+    border: 1px solid white;
+    background: #FFFFFF;
+    box-shadow: 0px 0px 10px rgba(0, 0, 0, 0.25);
+  }
+
+  &__identicon-border {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .icon-with-fallback__identicon-border {
+    position: absolute;
+  }
+
+  &:before {
```

Comment:
What is this section for? It wasn't clear even in testing what any of this did.

Reply:
doesn't seem to do anything, removed in 0ecda47b2